salt/metalk8s/addons/ui/config/metalk8s-shell-ui-config.yaml.j2

```diff
@@ -11,8 +14,9 @@ spec:
     responseType: "id_token"
     scopes: "openid profile email groups offline_access audience:server:client_id:oidc-auth-client"
   userGroupsMapping:
-    "admin@metalk8s.invalid":
-      - admin
+{%- for user in dex.spec.config.staticPasswords | map(attribute='email') %}
+    "{{ user }}": [admin]
```

Comment:
So, IIUC, in practice all Dex static users have the `admin` group mapped. Do we want another PR to filter MetalK8s pages based on this group? Also, to keep things clean, I would name this group `metalk8s:admin` (it's configurable anyway).